testprojects/src/python/python_distribution/ctypes_with_third_party/BUILD

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+## Third party integration testing.
+
+ctypes_compatible_c_library(
+  name='c_library',
+  sources=['some_math.h', 'some_math.c', 'src-subdir/add_three.h', 'src-subdir/add_three.c'],
+  ctypes_native_library=native_artifact(lib_name='asdf-c-tp'),
+)
+
+ctypes_compatible_cpp_library(
+  name='cpp_library_with_third_party',
+  sources=['some_more_math.hpp', 'some_more_math_with_third_party.cpp'],
+  ctypes_native_library=native_artifact(lib_name='asdf-cpp-tp'),
+  dependencies=[':rang', ':cereal']
+)
+
+python_dist(
+  name='ctypes_with_third_party',
```

Comment:
nit: rename to `python_dist_with_third_party_c` to be more descriptive?

Reply:
✅